packages/bson-bench/src/base.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+const {
+  Package,
+  BSONLib,
+  BenchmarkSpecification,
+  RunBenchmarkMessage,
+  BenchmarkResult,
+} = require(`${__dirname}/common`);
+/* eslint-enable @typescript-eslint/no-unused-vars */
+
+const { performance } = require("perf_hooks");
+const fs = require("fs");
+const process = require("process");
+const BSON = require("bson");
+
+/**
+ * @param result {BenchmarkResult}
+ * */
+function reportResultAndQuit(result) {
+  if (process.send) process?.send({ type: "returnResult", result });
+  process.disconnect();
+  process.exit(0);
```

Comment:
```suggestion
  process.exitCode = 0;
```

Node.js should just exit if we're done with our work, calling the exit function could hide future mistakes with resources not being cleaned up.